python/python/lancedb/query.py

```diff
@@ -1727,10 +1744,22 @@ class AsyncFTSQuery(AsyncQueryBase):
     def __init__(self, inner: LanceFTSQuery):
         super().__init__(inner)
         self._inner = inner
+        self._reranker = RRFReranker()
```

Comment:
We can probably allow this to be None by default, similar to how it's done for the synchronous API:
https://github.com/lancedb/lancedb/blob/50c30c5d34dab481ca00897ed9fb63aa8b033567/python/python/lancedb/query.py#L844